src/css/styles.css

```
@@ -449,6 +449,14 @@ html {
   vertical-align: middle;
 }
 
+.definition-title__tooltip-icon > img:focus {
+  outline-color: var(--search-focus-color);
+}
+
+.definition-title__tooltip-icon:focus {
+  outline-color: var(--search-focus-color);
+}
```

Perhaps use [`:focus-within`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) instead of having two rules?

```suggestion
.definition-title__tooltip-icon:focus, .definition-title__tooltip-icon:focus-within {
  outline-color: var(--search-focus-color);
}
```

---

I also tested this in Firefox, and the outline doesn't actually appear without setting its width and colour? So here's the revised suggestion:

```suggestion
.definition-title__tooltip-icon:focus, .definition-title__tooltip-icon:focus-within {
  outline: 1px solid var(--search-focus-color);
}
```